teleport-node/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Stage1: build from source
+FROM quay.io/cybozu/golang:1.15-focal AS build
+
+ARG TELEPORT_VERSION=6.0.1
+RUN git clone --depth 1 --branch v${TELEPORT_VERSION} https://github.com/gravitational/teleport && \
+    cd teleport && \
+    make build/teleport OS=linux
+
+# Stage2: setup runtime container
+FROM quay.io/cybozu/ubuntu-debug:20.04
+
+COPY --from=build /work/teleport/build/teleport /usr/local/teleport/bin/
+COPY --from=build /work/teleport/LICENSE /usr/local/teleport/
+COPY update-necocli.sh /etc/profile.d/
+
+RUN adduser --disabled-password --gecos "" --uid 10000 cybozu
+USER cybozu
+
+ENV PATH=$PATH:/usr/local/teleport/bin
+
+ENTRYPOINT [ "teleport", "start", "-c", "/etc/teleport/teleport.yaml" ]
```

Comment:
Where can I find the `/etc/teleport/teleport.yaml`?
If the file does not exist in neco-contianers, it is better to define this command in the neco-apps manifests.

Reply:
These args are the same as The upstream official container of teleport, so I think this configuration can avoid confusing users.